### PR TITLE
Integrate client into GUI

### LINKED ISSUE #71: Integrate Client in GUI

### PR DESCRIPTION
**Pull request by StanleyNeoh**
Closes #71

### CODE REVIEW

File: src/main/java/mycelium/mycelium/ui/client/ClientListCard.java [hunk 1]
```diff
@@ -25,37 +24,35 @@ public class PersonListCard extends UiPart<Region> {
...
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
-        person.getTags().stream()
-            .sorted(Comparator.comparing(tag -> tag.tagName))
-            .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        name.setText(client.getName().toString());
+        email.setText(client.getEmail().toString());
+        yearOfBirth.setText(client.getYearOfBirth().map(YearOfBirth::toString).orElse("No year of birth").toString());
+        source.setText(client.getSource().orElse("No source").toString());
+        mobileNumber.setText(client.getMobileNumber().map(Phone::toString).orElse("No mobile number").toString());
```
Comment (RubyNguyen07) [inline src/main/java/mycelium/mycelium/ui/client/ClientListCard.java:55]: Small thing but do we need toString() here?

Comment (StanleyNeoh) [reply src/main/java/mycelium/mycelium/ui/client/ClientListCard.java:55]: oh my bad, yea theres no need for toString(), will fix

**Review by RubyNguyen07 — CHANGES_REQUESTED** [review]
LGTM, just a little comment!

**Review by RubyNguyen07 — APPROVED** [review]
LGTM!

**Pull request merged**